apps/degen-web-ui/src/core/api/mongo/db.ts

```diff
@@ -20,6 +21,7 @@ export async function initDatabase(): Promise<{
   return {
     db,
     collections: {
+      discordUsers: await db.collection(AppConstants.DISCORD_USER_COLLECTION_NAME),
```

Comment:
we should discuss which collection makes sense to use here.

Do we already have a collection of Discord Users?

Can we add an `address` attribute for Wallet Address?